patterns/page-portfolio-home.php

```diff
@@ -226,7 +226,7 @@
 			<!-- /wp:paragraph -->
 
 			<!-- wp:paragraph {"fontSize":"small"} -->
-			<p class="has-small-font-size">email@example.com<br>+1 555 349 1806</p>
+			<p class="has-small-font-size"><?php esc_html_e( 'email@example.com', 'twentytwentyfive' ); ?><br><?php esc_html_e( '+1 555 349 1806', 'twentytwentyfive' ); ?></p>
```

Comment:
```suggestion
			<p class="has-small-font-size"><?php esc_html_e( 'email@example.com', 'twentytwentyfive' ); ?><br><?php echo esc_html_x( '+1 555 349 1806', 'Phone number', 'twentytwentyfive' ); ?></p>
```

Because phone number formats vary from country to country, it might be a good idea to make it clear that this is a phone number.